### PR TITLE
App migration ns7

### PR DESCRIPTION
**Pull request by Stell0**
Add migration helper for Acrobit mobile app
```
php /var/www/html/freepbx/rest/lib/appMigrationHelper.php [--restore]
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1103,3 +1103,14 @@ $config = [
 ];
 
 ```
+
+
+## Migrate mobile app extensions from old to new
+
+This helper change all mobile app extensions from old version to the new one
```
Comment (gsanchietti) [inline README.md:1110]:
```suggestion
This helper changes all mobile app extensions from old version to the new one
```

File: README.md [hunk 2]
```diff
@@ -1103,3 +1103,14 @@ $config = [
...
+
+```
+php /var/www/html/freepbx/rest/lib/appMigrationHelper.php
```
Comment (gsanchietti) [inline README.md:1113]: Are you sure SCL should not be enabled?

File: root/var/www/html/freepbx/rest/lib/appMigrationHelper.php [hunk 3]
```diff
@@ -0,0 +1,45 @@
+<?php
+
+#
+# Copyright (C) 2023 Nethesis S.r.l.
```
Comment (gsanchietti) [inline root/var/www/html/freepbx/rest/lib/appMigrationHelper.php:4]:
```suggestion
# Copyright (C) 2024 Nethesis S.r.l.
```

File: README.md (hunk 2 above)
Comment (Stell0) [reply README.md:1113]: yes, I used freepbx_db.conf that only loads db and not functions, and avoided the use of PHP function array_columns()

**Review by gsanchietti — APPROVED** [review]

**Pull request merged**